docs/Configuration.md

```diff
@@ -395,6 +395,75 @@ Jest's ESM support is still experimental, see [its docs for more details](ECMASc
 }
 ```
 
+### `fakeTimers` \[object]
```

Comment:
what happens if this is provided and the user calls `jest.useFakeTimers` without an argument. Is the config picked up at that point? Or is any programmatic usage isolated?

(docs should mention the answer to this question)

Reply:
Very good question. I tried to make it clear. Also it made me to take another look at the implementation. Apparently there was a bug. Fixed that.